.travis.yml

```diff
@@ -140,7 +140,7 @@ install:
       npm install;
 
       # for mjpeg stuff
```

Comment:
This line cannot be here.

Reply:
oh sorry! for some reason i thought when i did this the build ran and failed with an unrelated error. @mykola-mokhnach do you want to fix this in this PR or have me fix it independently?

Reply:
nvm, I'll try to fix it, because it is anyway necessary to check my changes